pulsar/resource_pulsar_namespace.go

```diff
@@ -310,6 +315,11 @@ func resourcePulsarNamespaceRead(d *schema.ResourceData, meta interface{}) error
 			replClusters[i] = cl
 		}
 
+		isAllowAutoUpdateSchema, err := client.GetIsAllowAutoUpdateSchema(*ns)
+		if err != nil {
+			return fmt.Errorf("ERROR_READ_NAMESPACE: GetMaxProducersPerTopic: %w", err)
```

Comment:
Should the error log line be
`fmt.Errorf("ERROR_READ_NAMESPACE: GetIsAllowAutoUpdateSchema: %w", err)`?